ctest-next/templates/test.c

```diff
@@ -0,0 +1,21 @@
+#include <stdbool.h>
```

Comment:
Could you add this to both the Rust and C files so the generated outputs get a tag?

```c
/* This file was autogenerated by ctest; do not modify directly */
{# ↑ Doesn't apply here, this is the template! #}
```